### PR TITLE
Publish Generative AI memos

### PR DESCRIPTION
**Pull request by clementbiron**
That changeset publish the Generative AI memos and ensure the content also appears in the French version of the website.

### CODE REVIEW

**Review by Ndpnt — APPROVED** [review]

File: content/_index.en.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: Open Terms Archive
 html_description: Open Terms Archive publicly records every version of the terms of digital services to enable democratic oversight.
 hero:
+  anchor: mission-statement
```
Comment (MattiSG) [inline content/_index.en.md:5]: Why is the anchor different in French and in English? 🤔

File: content/memos/soundraw-forbids-using-its-output-to-train-other-ai-models.md
Comment (MattiSG) [inline content/memos/soundraw-forbids-using-its-output-to-train-other-ai-models.md:1]: Let's remove this example for now, as it shows as the first entry in the list and is not representative of the value of a memo (content too short and the linked diff is trivial). We could bring it back when we have more recent ones.

File: content/_index.en.md (hunk 1 above)
Comment (clementbiron) [reply content/_index.en.md:5]: Because it is translated.

Comment (MattiSG) [reply content/_index.en.md:5]: I find this weird. “mission” is the same in French and in English, and having the same anchor means having the same document structure and enables easier linking across languages.

**Pull request merged**